### PR TITLE
HOSTEDCP-1221: add periodic job for multiarch hypershift e2e aws 4.17

### PR DESCRIPTION
**Pull request by Patryk-Stefanski**

### CODE REVIEW

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws

File: ci-operator/step-registry/hypershift/aws/run-e2e/external/hypershift-aws-run-e2e-external-commands.sh [hunk 1]
```diff
@@ -49,5 +49,6 @@ hack/ci-test-e2e.sh -test.v \
   --e2e.additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)" \
   --e2e.aws-endpoint-access=PublicAndPrivate \
   --e2e.external-dns-domain=service.ci.hypershift.devcluster.openshift.com \
+  "${EXTRA_ARGS}" \
```
Comment (sjenning) [inline ci-operator/step-registry/hypershift/aws/run-e2e/external/hypershift-aws-run-e2e-external-commands.sh:52]: This seems like too generic (dangerous) a mechanism for jobs to pass in raw flags.  What about a `MULTIARCH_ENABLED` flag the job can set and we do the flag translation in here, similar to `REQUEST_SERVING_COMPONENT_PARAMS`?

@stevekuznetsov what do you think?

Comment (stevekuznetsov) [reply ci-operator/step-registry/hypershift/aws/run-e2e/external/hypershift-aws-run-e2e-external-commands.sh:52]: Yeah, good call - the pattern used above would be a better option.

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws-multi

**Pull request merged**